src/commands/init.ts

```diff
@@ -17,21 +20,43 @@ export default class Init extends Command {
   ]
 
   static flags = {
-    version: Flags.string({ description: 'project version' })
+    version: Flags.string({ description: 'project version' }),
+    'from-hub': Flags.boolean({ description: 'clone a bundle from Entando hub' }),
+    'hub-url': Flags.string({ description: 'custom hub url' }),
   }
 
   public async run(): Promise<void> {
     const { args, flags } = await this.parse(Init)
 
-    const initializer = new InitializerService({
-      parentDirectory: process.cwd(),
-      name: args.name as string,
-      version: flags.version ?? DEFAULT_VERSION
-    })
+    const parentDirectory = process.cwd()
+    const name = args.name as string
+    const version = flags.version ?? DEFAULT_VERSION
 
-    // Displaying spinner while performing the scaffolding
-    CliUx.ux.action.start(`Initializing a new bundle project ${args.name}`)
-    await initializer.performScaffolding()
-    CliUx.ux.action.stop()
+    const options = { parentDirectory, name, version }
+
+    const filesys = new FSService({ parentDirectory, name })
+    const git = new GitService({ parentDirectory, name }, filesys)
+
+    const serviceTools = { git, filesys }
+
+    const initializer = new InitializerService(options, serviceTools)
```

Comment:
no need to group and wrap dependencies into other objects. make dependencies explicit

Reply:
removed and now exclusive to initializer service class